git/README.md

```diff
@@ -1,6 +1,7 @@
 [pull request]: https://help.github.com/articles/using-pull-requests
-[commit message]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
-[best practices]: https://chris.beams.io/posts/git-commit/
+[A Note About Git Commit Messages]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
```

Comment:
Complete nit, but I got confused by these references at the start of the file (which are not output), partly because the added URL refs are capitalized and the existing ones are lowercased.

Reply:
That's fair — I was trying to follow the established convention, but I personally find this style of linking more confusing than inline links. I'll change these three.